### PR TITLE
enable link verification in pipelines

### PR DESCRIPTION
**Pull request by KarishmaGhiya**

### CODE REVIEW

File: eng/pipelines/aggregate-reports.yml [hunk 1]
```diff
@@ -10,6 +10,11 @@ jobs:
     pool:
       vmImage: 'ubuntu-18.04'
     steps:
+
+    - template: ../common/pipelines/templates/steps/verify-links.yml
```
Comment (JimSuplizio) [inline eng/pipelines/aggregate-reports.yml:14]: If we're doing this as part of the Analyze step for every pipeline, does this really need to be in here? I don't think this is going to provide any value here.

File: eng/pipelines/templates/jobs/archetype-sdk-client.yml [hunk 2]
```diff
@@ -140,6 +140,10 @@ jobs:
           $(Build.SourcesDirectory)/eng/versioning/pom_file_version_scanner.ps1
         displayName: "Verify versions in POM files"
 
+      - template: ../../../common/pipelines/templates/steps/verify-links.yml
```
Comment (JimSuplizio) [inline eng/pipelines/templates/jobs/archetype-sdk-client.yml:143]: This should be lower in the list of verifications. We should verify POM files then README then CHANGELOGs and then do the link verification. It doesn't make sense that would we verify links before verifying the content of READMEs and CHANGELOGs.

File: eng/common/scripts/Verify-Links.ps1 [hunk 3]
```diff
@@ -125,7 +125,7 @@ function CheckLink ([System.Uri]$linkUri)
   }
   else {
     try {
-      $response = Invoke-WebRequest -Uri $linkUri
+      $response = Invoke-WebRequest -Uri $linkUri #-TimeoutSec 60
```
Comment (weshaggard) [inline eng/common/scripts/Verify-Links.ps1:128]: Are you seeing cases where this is taking a long time to timeout?

File: sdk/spring/README.md [hunk 4]
```diff
@@ -19,6 +19,7 @@ Below starters are available with latest release version. We recommend users to
 You can find them in [Maven Central Repository](https://search.maven.org/).
 The first three starters are also available in [Spring Initializr](http://start.spring.io/). 
 
+<<<<<<< HEAD
```
Comment (weshaggard) [inline sdk/spring/README.md:22]: Bad merge.

**Comment by KarishmaGhiya** [comment]
Closing this in favor of https://github.com/Azure/azure-sdk-for-java/pull/13989

**Pull request closed without merging**